### ISSUE TITLE
Refused to execute script from 'https://path/libflac4-1.3.2.min.js' because its MIME type ('text/plain') is not executable

### ISSUE THREAD

**Issue by bramhechatejas** [mmig/libflac.js#14]
I have used libflac v 2-1.3.2 for encoding purpose. When I try running encoder deployed at dev server, chrome gives me this error. 

```
Refused to execute a script from 'https://path/libflac4-1.3.2.min.js' because its MIME type ('text/plain') is not executable
Uncaught DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://path/libflac4-1.3.2.min.js' failed to load.
```

Could you please check & let me know why it's refusing to run this script?

**Comment by russaa** [comment]
I don't think this is a problem specific to `libflac.js` but with the dev server:  

after a quick search, the following thread seems to describe a similar problem related to `Windows` in combination with (dev) servers that have a strict handling of the MIME type (which is specified in the `Windows` registry), see
https://github.com/janeczku/calibre-web/issues/896

and the referenced elaboration:
https://github.com/jupyterlab/jupyterlab/issues/6098#issuecomment-474098196
and the solution/fix (of course specific to that server/library)
https://github.com/jupyter/notebook/pull/4468
